src/Sentry.NLog/Sentry.NLog.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageTags>$(PackageTags);Logging;NLog</PackageTags>
+    <PackageId>Sentry.NLog</PackageId>
+    <AssemblyName>Sentry.NLog</AssemblyName>
+    <RootNamespace>Sentry.NLog</RootNamespace>
+    <Description>Official NLog integration for Sentry - Open-source error tracking that helps developers monitor and fix crashes in real time.</Description>
+  </PropertyGroup>
+  
+  <ItemGroup>
+    <PackageReference Include="NLog" Version="4.6.1" />
```

Comment:
Is there a reason for the specific version `4.6.1`? We should aim to target the lowest possible here and let the application roll forward. Would `4.6.0` be a possibility?

Reply:
NLog 4.5.0 would also be possible.